test/unit/parachain/escrow.test.ts

```diff
@@ -380,4 +388,11 @@ describe("Escrow", () => {
         assert.equal(result.apy.round(2).toString(), apy.round(2).toString());
         assert.isTrue(result.apy.gt(0), `${result.apy.toString()} not greater than 0`);
     });
-})
+
+    it("should return the total stake as a governance token amount", async () => {
```

Comment:
@theodorebugnet @savudani8 does this have enough value as a unit test? We don't have any amount staked in the local parachain so 0 is what we would expect, but not sure if it's covering enough ground.

Reply:
Not really, this should be moved to the integration tests. After a user `createLock`s, the test should ensure the stake is equal to the locked amount.

Reply:
Glad you said that :) couldn't really see what the point of it was, but thought I might be missing something! I'll move it over to the integration tests.